### ISSUE TITLE
UnicodeDecodeError whit non-ascii character in rtichoke.prompt using rtichoke-0.2.0.dev0 on python 2.7

### ISSUE THREAD

**Issue by dcangst** [randy3k/rtichoke#52]
Works no problem in rtichoke 1.9.0/python2.7 or rtichoke-0.2.0.dev0/python3.6
maybe not a bug worth worrying about, but maybe remove python2 support?

this is the offending line in my .Rprofile: rtichoke.prompt = "\033[0;34m🐰 >\033[0m ". (I like my little rabbit when i do my data analysis ;))

i'm running macOS 10.13.4, R 3.5.0 (x86_64-apple-darwin15.6.0 (64-bit))

Traceback:
```
Traceback (most recent call last):
  File "/usr/local/bin/rtichoke", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/rtichoke/__init__.py", line 54, in main
    RtichokeApplication(r_home).run(options)
  File "/usr/local/lib/python2.7/site-packages/rtichoke/rtichokeapp.py", line 163, in run
    session_initialize(session)
  File "/usr/local/lib/python2.7/site-packages/rtichoke/prompt.py", line 165, in session_initialize
    prompt = interface.get_option("rtichoke.prompt", None)
  File "/usr/local/lib/python2.7/site-packages/rapi/interface.py", line 383, in get_option
    ret = rcopy(Rf_GetOption1(Rf_install(key.encode())))
  File "/usr/local/lib/python2.7/site-packages/rapi/dispatch.py", line 48, in __call__
    return func(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/rapi/interface.py", line 352, in rcopy
    return rcopy(T, s)
  File "/usr/local/lib/python2.7/site-packages/rapi/dispatch.py", line 48, in __call__
    return func(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/rapi/interface.py", line 261, in rcopy
    return CHAR(STRING_ELT(s, 0)).decode()
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 9: ordinal not in range(128)
```

**Comment by randy3k** [comment]
It should definitely be fixed. Thanks for the report.

**Comment by randy3k** [comment]
Fixed now with rapi 0.0.6.

<img width="521" alt="screen shot 2018-05-14 at 12 36 23 pm" src="https://user-images.githubusercontent.com/1690993/40010666-6e323df4-5773-11e8-8433-870629dc8001.png">
